app/assets/javascripts/miq_application.js

```diff
@@ -1272,14 +1272,19 @@ function miqSelectPickerEvent(element, url, options) {
   $('#' + element).on('change', function() {
     var selected = $(this).val();
     var finalUrl = url + (firstarg ? '?' : '&') + element + '=' + escape(selected);
-    if($(this).attr('data-miq_sparkle_on') == 'true') {
+    var is_sparkle_on = $(this).attr('data-miq_sparkle_on') == 'true'
+    var is_sparkle_off = $(this).attr('data-miq_sparkle_off') == 'true'
+
+    if(is_sparkle_on) {
```

Comment:
(minor, style) .. add a space between `if` and `(` please.. the same for is_sparkle_off..